pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -196,8 +204,8 @@ func (dswp *desiredStateOfWorldPopulator) findAndAddNewPods() {
 
 	processedVolumesForFSResize := sets.NewString()
 	for _, pod := range dswp.podManager.GetPods() {
-		if dswp.isPodTerminated(pod) {
-			// Do not (re)add volumes for terminated pods
+		if dswp.isPodTerminating(pod) {
```

Comment:
Can you explain more about why you want `isPodTerminating` to replace `isPodTerminated` instead of `isPodTerminating || isPodTerminated`?

And if we are replacing `isPodTerminated`, do we still need that function? Is it used anywhere else?